pylustrator/QtGuiDrag.py

```diff
@@ -192,7 +191,7 @@ def show(hide_window: bool = False):
         if setting_use_global_variable_names:
             setFigureVariableNames(figure)
         # get the window
-        #window = _pylab_helpers.Gcf.figs[figure].canvas.window_pylustrator
+        # window = _pylab_helpers.Gcf.figs[figure].canvas.window_pylustrator
```

Comment:
I think there is value to have code lines that are commented out without the space to distinguish them from "real" comments which are annotations